migrate/live-migration/live-migration-from-timescaledb.md

```diff
@@ -158,5 +137,7 @@ Application downtime ends here.
 Once you are confident with the data validation, the final step is to configure
 your applications to use the target database.
 
+<LiveMigrationDockerCleanup />
```

Comment:
It is better to recommend cleanup only when the are confident on the migration is successful. Premature cleanup would hinder the ability to resume if they find anything missing in the target.

Reply:
There is a mention of "Once you are confident ..." before we recommend the cleanup. Additionally, I have added below text to convey the message more clearly.

```text
It's important to note that executing the`clean` command means you cannot resume the live migration.
```